hack/lib/common.bash

```diff
@@ -31,15 +31,9 @@ function timeout {
   while eval $*; do
     seconds=$(( seconds + interval ))
     logger.debug "Execution failed: ${*}. Waiting ${interval} sec ($seconds/${timeout})..."
-    if [[ "${LOG_LEVEL}" != 'DEBUG' ]]; then
-      echo -n '.'
-    fi
```

Comment:
I actually kinda like the dots. :smile: Are you just removing them due to the -x noise? Can we set +x in timeout instead maybe?

Reply:
I can bring the loglevels back. I just removed the entire loglevel setting + only showing logs of a certain level. You'd now also see debug logs locally. Want me to roll that back? The `x` noise wouldn't be affected because I override the entire functions for CI anyway.

Reply:
I may have misunderstood then. As long as I see *something*, whether dot or log message, I'm good. I just want to know something is doing something.

Reply:
Yeah the log messages will now be printed on each run, giving you the failure reason.